### DISCUSSION TITLE
Welcome to hackpack-web Discussions!

### DISCUSSION THREAD

**Discussion by kapravel** [kapravel/hackpack-web#10] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by crollins18** [comment]
Hello all. I am Caleb Rollins! I am studying computer science and I am currently looking into the cybersecurity concentration that the csc department has. I like doing web challenges but I am trying to learn more about binary exploitation. I enjoy hackpack not only because of the new skills I learn, but for the community of like minded students that is fostered. Check out my website [crollins.io](http://crollins.io) for more info about myself.
